dev/source/docs/code-overview-scheduling-your-new-code-to-run-intermittently.rst

```diff
@@ -15,65 +15,44 @@ scheduler.  This can be done by adding your new function to the
 `scheduler_tasks <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp#L91>`__
 array in
 `Copter.cpp <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp>`__. 
-Note that there are actually two task lists, `the upper list <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp#L788>`__
-is for high speed CPUs (i.e. Pixhawk) and the `lower list <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp#L856>`__
-is for slow CPUs (i.e. APM2).
 
-Adding a task is fairly simple, just create a new row in the list
-(higher in the list means high priority).  The first column holds the
-function name, the 2nd is a number of 2.5ms units (or 10ms units in case
-of APM2).  So if you wanted the function executed at 400Hz this column
-would contain "1", if you wanted 50Hz it would contain "8".  The final
-column holds the number of microseconds (i.e. millions of a second) the
-function is expected to take.  This helps the scheduler guess whether or
-not there is enough time to run your function before the main loop
-starts the next iteration.
+Helpers are provided to create entries in the scheduler table; see the documentation referenced above.
```

Comment:
Thanks for this.  I think we should fill this section out a bit more though.  We should tell the user to add a new row to the scheduler task list and I actually think we should describe the helpers a little bit.  Also the "documentation referenced above" does not mention FAST_TASK.